src/sentry/static/sentry/app/components/contextPickerModal.tsx

```diff
@@ -368,9 +374,15 @@ const ContextPickerModalContainer = createReactClass<ContainerProps, ContainerSt
   },
 
   render() {
+    const {projectSlugs} = this.props; // eslint-disable-line react/prop-types
```

Comment:
Let's add the propType instead of disabling the rule, we're keeping propTypes around until we are all converted to TS.

Reply:
I was considering it, but then I noticed that we ditched all proptypes when converting this component to typescript, because it's not consumed by any `jsx` file anymore.